packages/augur-tools/src/flash/flash.ts

```diff
@@ -60,7 +60,11 @@ export class FlashSession {
     public accounts: Account[],
     public network?: string,
     public config?: SDKConfiguration,
-  ) {}
+  ) {
+    if(config) {
+      config.useWarpSync = false;
+    }
+  }
```

Comment:
Better to do this in `cli.ts` because we mess with config there already.

Reply:
what about adding something to `packages/augur-artifacts/src/configuration.ts` to turn it on so it'll be off by default

Reply:
It can be set to `false` in the default config then anywhere that wants to use it can set it to true with mergeConfig.